arelle/CntlrCmdLine.py

```diff
@@ -753,9 +758,9 @@ def monitorParentProcess():
         if options.outputAttribution:
             self.modelManager.outputAttribution = options.outputAttribution
         self.modelManager.validateTestcaseSchema = options.validateTestcaseSchema
-        if options.internetConnectivity == "offline":
+        if options.internetConnectivity == "offline" or options.disableConfigCache:
```

Comment:
I know I'm the one who said offline mode should be triggered by this setting, but if we use `--disablePersistentConfig` then I think it would be better to not change the offline mode behavior and instead require users to pass both flags (beyond the issue that triggered this, I can see a desire to not persist config settings while still having a dynamic cache)